app/javascript/groceries/store.ts

```diff
@@ -177,13 +172,11 @@ export const useGroceriesStore = defineStore('groceries', {
         }
       };
 
-      interface StoresResponse {
-        own_stores: Array<Store>;
-        spouse_stores: Array<Store>;
-      }
```

Comment:
(Nonessential, unrelated change. This is only used in one place; it seemed more appropriate to inline.)